buildSrc/build.gradle.kts

```diff
@@ -1,17 +1,15 @@
 plugins {
-    `kotlin-dsl`
+    kotlin("jvm").version("1.3.61")
 }
 
-val kotlinVersion = "1.3.50"
-
 repositories {
     google()
     jcenter()
 }
 
 dependencies {
+    implementation(kotlin("stdlib"))
```

Comment:
@martinbonnin I'm not sure why this implementation is required? 🧐

Reply:
Technically, it should be working without because okhttp declares the stdlib as an api dependency: https://github.com/square/okhttp/blob/master/okhttp/build.gradle#L26

But `buildSrc` also uses the kotlin-stdlib itself so it's a good practice to declare it to make dependencies explicit.